struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter-close.ftl

```diff
@@ -22,49 +22,49 @@
 <script type='text/javascript'>
 jQuery(document).ready(function () {
   <#if parameters.valueWidget?if_exists != "">
-	jQuery("#${parameters.id?html}").val("${parameters.valueWidget?html}");
+	jQuery("#${parameters.i}").val("${parameters.valueWidget}");

```

Comment:
`i`? shouldn't be `id`?

Reply:
Will look into it when i get home, probably already fixed it localy and forgot to commit it.

Reply:
fixed